scripts/electronpublish.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const util = require('util');
+const rimraf = util.promisify(require('rimraf'));
+const archiver = require('archiver');
+const baseDir = path.join(__dirname, '..');
+const buildDir = path.join(baseDir, 'build');
+const distDir = path.join(baseDir, 'dist');
+
+(async () => {
+  await rimraf(distDir);
+  await mkdirp(distDir);
```

Comment:
Is this actually necessary?
As far as I can tell, `fs.createWriteStream` would just overwrite existing files.

For cleaning out `dist` the following might be sufficient.
```js
if (fs.existsSync(distDir)) {
  (await fs.promises.readdir(buildDir)).forEach((oldZip) => {
    await fs.promises.unlink(oldZip);
  });
} else {
  await fs.promises.mkdir(distDir);
}